src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/EventSource.cs

```diff
@@ -347,7 +347,12 @@ public static string GetName(Type eventSourceType)
         /// <param name="assemblyPathToIncludeInManifest">The manifest XML fragment contains the string name of the DLL name in
         /// which it is embedded.  This parameter specifies what name will be used</param>
         /// <returns>The XML data string</returns>
-        public static string? GenerateManifest(Type eventSourceType, string? assemblyPathToIncludeInManifest)
+        public static string? GenerateManifest(
+#if !ES_BUILD_STANDALONE
+            [DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicNestedTypes | DynamicallyAccessedMemberTypes.PublicMethods | DynamicallyAccessedMemberTypes.NonPublicMethods)]
```

Comment:
The way I was thinking about this was to put a const field:

```C#
private const DynamicallyAccessedMemberTypes ManifestMemberTypes = DynamicallyAccessedMemberTypes.PublicNestedTypes | DynamicallyAccessedMemberTypes.PublicMethods | DynamicallyAccessedMemberTypes.NonPublicMethods;
```

And then in the 3 attributes that need this, use the const field.

The advantages are:

1. It reduces duplication.
2. It shrinks the code lines, so they fit on a screen.